src/main.js

```diff
@@ -75,7 +76,7 @@ function update () {
   const htmlForPreview = createHtml({ html, js, css })
   $('iframe').setAttribute('srcdoc', htmlForPreview)
 
-  debounce(updateHashedCode, 1000)({ html, css, js })
+  debounce(updateHashedCode, MS_UPDATE_HASH_DEBOUNCED_TIME)({ html, css, js })
```

Comment:
You're creating a debounced function every time that update is called. This is wrong, because the control of the internal timer of the debounced function it's different every update time. You need to create a ```debouncedUpdateHash``` (in line ```60```) and then use in the line ```79```

Reply:
Hi @javigaralva 

Thanks for it.  Is this what you mean?


```js
const debouncedUpdateHash = debounce(updateHashedCode, MS_UPDATE_HASH_DEBOUNCED_TIME)

function update () {
  const html = htmlEditor.getValue()
  const css = cssEditor.getValue()
  const js = jsEditor.getValue()

  const htmlForPreview = createHtml({ html, js, css })
  $('iframe').setAttribute('srcdoc', htmlForPreview)

  debouncedUpdateHash({ html, css, js })
}
```

Reply:
Thank you for the review. I have fixed it